libbindgen/tests/headers/whitelist-namespaces-basic.hpp

```diff
@@ -0,0 +1,10 @@
+// bindgen-flags: --enable-cxx-namespaces --whitelist-type '.*Helper'
+
+namespace outer {
+  namespace inner {
+    struct Helper {};
+  }
+  struct Test {
```

Comment:
It isn't, though, right? Note there are two expectations file, one being your test case, and other white listing Helper.